clients/src/main/java/org/apache/kafka/common/requests/ProduceResponse.java

```diff
@@ -131,6 +131,7 @@ public Map<Errors, Integer> errorCounts() {
     public static final class PartitionResponse {
         public Errors error;
         public long baseOffset;
+        public long lastOffset;
```

Comment:
i always assumed that there was a 1-1 mapping between the json (ProduceResponse.json) and the internal representation. i'm guessing we don't have to change the actual protocol since we only use this value internally?

Reply:
Yeah, it is usually the case. Ss we don't use the auto-generated classes here (yet), we can do this.